### PR TITLE
Cuda8

### PR DESCRIPTION
**Pull request by mosamlife**

### CODE REVIEW

**Comment by MariusVanDerWijden** [comment]
I think the master should use CUDA 9, because it is way faster (and we support volta)

**Comment by jean-m-cyr** [comment]
Did I delete sm 70!!! Didn't mean to. Needs to be included

**Comment by MariusVanDerWijden** [comment]
I deleted it, otherwise it wouldn't work... you can't compile with cuda 8.0 to sm_70 (volta)

**Comment by jean-m-cyr** [comment]
I need to handle it in cmake then. We need Volta.

**Comment by jean-m-cyr** [comment]
I suppose it's reasonable to expect an N-1 version compiler to work.

**Comment by MariusVanDerWijden** [comment]
Thats why I think we should use two branches, one legacy for cuda 8.0 and the master for cuda 9.0

**Comment by jean-m-cyr** [comment]
Need to run that by @chfast

**Comment by jean-m-cyr** [comment]
With two branches, we often have to back port fixes. There are down sides.

**Comment by MariusVanDerWijden** [comment]
I know... I wouldn't support cuda 8.0 anyway. Since 9.0 gives us a slight performance boost

**Comment by chfast** [comment]
We don't support CUDA 8 I believe. That means users have to upgrade Nvidia drivers. Are there any hardware that stops working because of that? (I mean Nvidia drivers not providing CUDA9 runtime)

**Comment by MariusVanDerWijden** [comment]
only sm_2x...

File: libethash-cuda/CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +19,6 @@ else()
 		"-gencode arch=compute_60,code=sm_60"
 		"-gencode arch=compute_61,code=sm_61"
 		"-gencode arch=compute_62,code=sm_62"
-		"-gencode arch=compute_70,code=sm_70"
```
Comment (chfast) [inline libethash-cuda/CMakeLists.txt]: ?

Comment (jean-m-cyr) [reply libethash-cuda/CMakeLists.txt]: @chfast Not usable hardware. I guess that pretty well settles it. Cuda 9 only support.

Comment (jean-m-cyr) [reply libethash-cuda/CMakeLists.txt]: And yes, we keep sm 70. That was just to make cuda 8 work

**Comment by jean-m-cyr** [comment]
Closing

**Pull request closed without merging**